src/test/java/com/meilisearch/integration/SettingsTest.java

```diff
@@ -210,12 +236,12 @@ public void testUpdateSynonymsSettings() throws Exception {
         newSynonymsSettings.put("wow", new String[] {"world of warcraft"});
 
         index.waitForTask(index.updateSynonymsSettings(newSynonymsSettings).getUid());
-        Map<String, String[]> updatedRankingRuleSettings = index.getSynonymsSettings();
+        Map<String, String[]> updatedSynonymsSettings = index.getSynonymsSettings();
 
-        assertEquals(newSynonymsSettings.size(), updatedRankingRuleSettings.size());
-        assertEquals(newSynonymsSettings.keySet(), updatedRankingRuleSettings.keySet());
-        assertNotEquals(synonymsSettings.size(), updatedRankingRuleSettings.size());
-        assertNotEquals(synonymsSettings.keySet(), updatedRankingRuleSettings.keySet());
+        assertEquals(newSynonymsSettings.size(), updatedSynonymsSettings.size());
```

Comment:
It seems just a harmful change, but when you are reviewing is hard to understand all the contexts of a PR. In this case, this refactor could be a different PR pointing directly to the `main` ;)